src/codelabs/null-safety.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Null safety codelab
+title: Null safety tutorial
 description: Learn about and practice writing null-safe code in DartPad!
 js: [{url: 'https://dartpad.dev/inject_embed.dart.js', defer: true}]
 ---
 {% assign useIframe = false -%}
 <?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /(^|\n)*\/\/\s+ignore:[^\n]+\n/$1/g; /(\n[^\n]+) *\/\/\s+ignore:[^\n]+\n/$1\n/g"?>
 <?code-excerpt plaster="none"?>
 
-This codelab teaches you about Dart’s [null-safe type system](/null-safety),
+This tutorial teaches you about Dart’s [null-safe type system](/null-safety),
 which was introduced in Dart 2.12. When you opt into null safety,
 types in your code are non-nullable by default,
 meaning that values can’t be `null` unless you say they can be.
```

Comment:
**suggestion:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
This tutorial explains Dart’s [null-safe type system](/null-safety).
Dart 2.12 introduced null safety and Dart 3 made it the default setting.
With null safety, types in your code default to non-nullable.
This means that values can’t be `null` unless you say they can be.
```